vignettes/RunningCohortDiagnostics.Rmd

```diff
@@ -126,7 +118,36 @@ The resulting cohort table should include the columns:
 
 `cohort_definition_id`, `subject_id`, `cohort_start_date`, `cohort_end_date`
 
+### Inclusion rules
+We have the option to also generate inclusion rule statistics while the cohorts are instantiated (recommended).
+If we want to do this, we need to provide a folder where the inclusion rule statistics will be stored for later use.
 
+```{r}
+inclusionStatisticsFolder <- "inclusionStats"
+```
+We can then use the cohort generator package to create the inclusion rule statistics for our cohorts.
+```{r eval = FALSE}
+inclusionRuleNames <- CohortGenerator::insertInclusionRuleNames(connectionDetails = connectionDetails,
+                                                                cohortDefinitionSet = cohortDefinitionSet,
+                                                                cohortDatabaseSchema = cohortDatabaseSchema,
+                                                                cohortInclusionTable = cohortTableNames$cohortInclusionTable)
+
+CohortGenerator::exportCohortStatsTables(connectionDetails = connectionDetails,
+                                         cohortDatabaseSchema = cdmDatabaseSchema,
+                                         cohortTableNames = cohortTableNames,
+                                         cohortStatisticsFolder = inclusionStatisticsFolder,
+                                         incremental = FALSE)
+
+CohortGenerator::dropCohortStatsTables(connectionDetails = connectionDetails,
```

Comment:
I'm wondering about adding the dropChortStatsTable() call in this example.  It's good to have a cleanup function but, is the expected workflow is to generate -> export -> drop tables?  I was expecting more of a multi-generate and when you're done with your study you can drop the tables.

Reply:
This is reasonable, it isn't strictly necessary for this guide. I'll remove it